dokka-integration-tests/gradle/build.gradle.kts

```diff
@@ -22,10 +22,6 @@ dependencies {
 val aggregatingProject = gradle.includedBuild("dokka")
 
 tasks.integrationTest {
-    // pass the property to a test fork
-    project.findProperty("org.jetbrains.dokka.experimental.tryK2")
-        ?.let { systemProperty("org.jetbrains.dokka.experimental.tryK2", it) }
-
```

Comment:
I moved this configuration to the integration test convention plugin. The Maven/CLI integration tests might not use it, but to be it made sense to move as much config as possible into convention plugins to keep the project build scripts smaller.